tests/epics/adaravis/test_aravis.py

```diff
@@ -105,7 +107,7 @@ async def test_can_collect(
     assert stream_resource["parameters"] == {
         "dataset": "/entry/data/data",
         "swmr": False,
-        "multiplier": 1,
+        "shape": (one_shot_trigger_info.frames_per_event, 10, 10),
```

Comment:
Like I mentioned before, I don't know if we need `shape` in the resource doc since we can retrieve it from the data key outputted by `describe`? @coretl thoughts?

Reply:
yeah, don't need it in the resource doc if it is in the descriptor